app/src/main/res/xml/preferences.xml

```diff
@@ -31,12 +31,17 @@
             app:useSimpleSummaryProvider="true"/>
     </PreferenceCategory>
     <PreferenceCategory
-        android:title="@string/sensors">
+        android:title="@string/entities">
         <Preference
             android:key="sensors"
             android:icon="@drawable/leak"
             android:title="@string/sensor_title"
             android:summary="@string/sensor_summary"/>
+        <Preference
+            android:key="nfc_tags"
```

Comment:
NFC tags are not an entity, they only fire an event to HA.

Reply:
What pref category name you'd recommend?

Reply:
Umm I think it made sense to keep it under Other?

Reply:
Ok, I'll revert this